src/main/java/com/volasoftware/tinder/services/UserService.java

```diff
@@ -116,6 +116,20 @@ public User getLoggedUser(){
         .orElseThrow(() -> new UserDoesNotExistException("User not found!"));
     }
 
+    public void generateNewPassword(String email) throws IOException, MessagingException {
```

Comment:
Actually the method creates new password, change the existing one of the user and send email, not exactly what the method name says